src/ethereum/prague/vm/precompiled_contracts/bls12_381/__init__.py

```diff
@@ -463,18 +453,17 @@ def bytes_to_FQ2(
     c_0 = int.from_bytes(data[:64], "big")
     c_1 = int.from_bytes(data[64:], "big")
 
-    if c_0 >= P:
+    if c_0 >= FQ.field_modulus:
         raise InvalidParameter("Invalid field element")
-    if c_1 >= P:
+    if c_1 >= FQ.field_modulus:
         raise InvalidParameter("Invalid field element")
 
-    if optimized:
-        return OPTIMIZED_FQ2((c_0, c_1))
-    else:
-        return FQ2((c_0, c_1))
+    return FQ2((c_0, c_1))
 
 
-def bytes_to_G2(data: Bytes) -> Point2D:
+def bytes_to_g2(
```

Comment:
A nit here. We should have a consistent naming convention for the functions `bytes_to_G2` instead of `bytes_to_g2` (all small letters). Elsewhere in the module, there is `FQ2_to_bytes`. Irrespective of which way we name, we should be consistent

Reply:
Yeah. All lowercase sounds good to me.